app/app.R

```diff
@@ -42,7 +42,7 @@ country_colors <- paste0("color:",c('#4B73B3', "#8BCFA5","#C02C4A", '#FED47F','#
 ### USER INTERFACE ###
 
 ui <- navbarPage("GROW Database Explorer",
-                 theme = bslib::bs_theme(
+                 theme = bs_theme(
```

Comment:
absolute nitpick: in general, these prefixes are unnecessary. There are a few instances in this and in the src script where these exist.